src/components/DetailedTask.jsx

```diff
@@ -187,7 +202,7 @@ function DetailedTask(props) {
                 <h3 className="detailedTitle">{props.task.title}</h3>
                 <button className="close" onClick={props.closeCard}>x</button>
                 <p className="user">Created by: {props.task.user}</p>
-                <form className ="editTaskForm" onSubmit={(e) => props.handleEditTask(e,props.task.taskId, detailedTask)}>
+                <form className ="editTaskForm" onSubmit={(e) => {props.handleEditTask(e,props.task.taskId, detailedTask); handleUpdate(e)}} >
```

Comment:
this function can be put on new lines to make it more readable since it is more than one command